src/Microsoft.AspNetCore.OData.Authorization/DefaultScopesEvaluator.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.OData.Authorization
+{
```

Comment:
Some comment about what `DefaultScopesEvaluator` does/allows

Reply:
In retrospect, I think the `DefaultScopesEvaluator` is probably not a good idea. It's supposed to be a stand-in for when a particular restriction is missing from the model, so this evaluator assumes that it's allowed if it's not restricted (that's why it always return true). An alternative would be return `null` instead of a `DefaultScopesEvaluator` and just perform null-checks.

Reply:
But I've added a doc comment.